### ISSUE TITLE
A11y_.Net Core_Winforms_MainForm_Common Controls1_ Date Picker Section_High Contrast : In High contrast themes , The Check box focus and Date picker Icon is not properly visible.

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [dotnet/winforms#9234]
### .NET version

.NET-SDK-8.0.100-preview.4.23260.5

### Did it work in .NET Framework?

No

### GitHubTags:
#A11yMAS;#A11yTCS;#BM_.NET Core[Winforms]-Win32-May2023;#.NET Core;#A11ySev3;#WCAG4.3.1;#DesktopApp;#Win11;#Benchmark;

### Pre Requisite:

1. Turn on Contrast modes in System settings
2. Choose any themes

### Actual:
In All High Contrast Themes , the Check box focus and Date picker Icon is not properly visible.

Note : This issue is present in all the high contrast themes.

### Expected:
The Keyboard focus on check box and Date picker icon should be properly visible with all the high contrast themes.

### User Impact:
Users who use high contrast themes for Navigating through out the UI will face difficulty in Knowing the proper focus and proper control

### Steps to reproduce

1. Install and Open Winforms App
2. Main Form screen would appear.
3. Tab till "Common Controls_1: Testing the controls under common controls Tab" and Hit enter.
4. Navigate to "Date picker" Section
5. Observe with High contrast themes that the keyboard focus and Date picker Icon is not properly visible.

### Attachment : 
![High Contrast](https://github.com/dotnet/winforms/assets/85863895/3253ce85-53dd-4abd-bfc3-ec1da3843846)

**Comment by Ashley-Li** [comment]
Duplicate to #7705 , it was treated as won't fix. For more information, please see #7705 comments.

**Comment by Ashutoshsahoo164** [comment]
Hi @Ashley-Li , Can you check keyboard focus on the Checkbox using High contrast?

**Comment by merriemcgaw** [comment]
@Tanya-Solyanik confirming that this is Common Controls? If so I'll add this to the list of exceptions I need to request.

**Comment by Tanya-Solyanik** [comment]
@merriemcgaw - this had been fixed by windows in 22H2 - https://github.com/dotnet/winforms/issues/7705#issuecomment-1250011124. We don't need an exception.
@Ashutoshsahoo164  - we support only the latest version of windows, please test at least on 22H2 (if you don't have access to private builds).

**Comment by Ashutoshsahoo164** [comment]
Hi @Tanya-Solyanik I have Windows 23H2 version but still it is reproducing at my side.

**Comment by Tanya-Solyanik** [comment]
@merriemcgaw  - @Ashley-Li had verified that this icon is displayed for supported OS set to win10 and is not displayed when supported OS is set to older versions. However this is still external to winforms and is a Sev3, thus closing this bug

**Comment by Ashutoshsahoo164** [comment]
GithubTags:#Closed;
